hosts/signed-int16/home/reisen/home.nix

```diff
@@ -7,15 +7,60 @@
   ...
 }: {
   imports =
-    [
+    [ 
       (path + /modules/shared/home/reisen/programs/dev/vim.nix)
     ]
     ++ (import (path + /modules/shared/home/reisen/programs/utils));
   home = {
     username = "reisen";
     packages = with pkgs; [
+      # Audio
+      tenacity
+      pavucontrol
+      qpwgraph
+      
+      # BROWSERS
+      ungoogled-chromium
+      firefox
+      librewolf
+
+      # Graphics
+      krita
+      gimp
+      inkscape
+      kdenlive
+
+      # Media
+      quodlibet
+      vlc
+
+      # Dev
+      lazygit
+      vscodium-fhs
```

Comment:
vscodium can be declared with home-manager
```nix
programs.vscode = {
    enable = true;
    package = pkgs.vscodium-fhs;
};
```

Reply:
did that also configured vscode itself a little with it